lib/src/command/moronepo_command.dart

```diff
@@ -1,20 +1,57 @@
 import 'package:args/command_runner.dart';
+import 'package:args/src/arg_results.dart';
 
 abstract class MoronepoCommand<T> extends Command<T> {
   MoronepoResults get moronepoResults => MoronepoResults(
         workingDirectory: _fromGlobalResults("working-directory"),
-        projectName: _fromGlobalResults("project"),
+        projectFilters: ProjectFilters.from(globalResults),
       );
 
-  R _fromGlobalResults<R>(String name) => globalResults[name] as R;
+  R _fromGlobalResults<R>(String name) => _fromResults(globalResults, name);
 }
 
 class MoronepoResults {
   final String workingDirectory;
-  final String projectName;
+  final ProjectFilters projectFilters;
 
   MoronepoResults({
     this.workingDirectory,
-    this.projectName,
+    this.projectFilters,
   });
 }
+
+class ProjectFilters {
```

Comment:
Separate file?  `ProjectFilters` is something that I will never been looking in this file.